src/components/carousel-section-immobile/carousel-section-immobile.tsx

```diff
@@ -0,0 +1,21 @@
+import { CarouselPhotos } from '../carousel-phostos/carousel-photo';
+
+const ImmobileData = {
```

Comment:
lembrar do padrao de nomenclatura
```suggestion
const immobileData = {
```